gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ProjectInfo.java

```diff
@@ -31,7 +31,6 @@
 
 /**
  * A Google Cloud Resource Manager project metadata object.
- * 
```

Comment:
If you want to make checkstyle happy add a <p> after a new line (no new line after the <p>).
Also, State's ACTIVE and LIFECYCLE_STATE_UNSPECIFIED are missing a '.' in their javadoc.

Maybe after this PR run google checkstyle on all module files.